pkgs/applications/office/pyspread/default.nix

```diff
@@ -7,12 +7,12 @@
 , makeDesktopItem
 }:
 
-python3.pkgs.buildPythonApplication rec {
+python3.pkgs.buildPythonApplication (finalAttrs: {
   pname = "pyspread";
   version = "2.0.2";
 
   src = fetchPypi {
-    inherit pname version;
+    inherit (finalAttrs) pname version;
```

Comment:
Using pname from finalAttrs is potentially dangerous here because it breaks if you add a suffix.